Changelogs for Digital Twin Manager Plugins 3.1.1 (#3659)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1022-4-0-add-confirmation-message-for-group-deletion-in-cockpit-ctm-2311.md

---
date: ""

> title: "Add confirmation message for group deletion"

Reviewer: ```suggestion
title: "Added confirmation message for group deletion"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2311"
version: "1022.4.0"
---

> Previously, while deleting a group, the radio buttons were displayed with an empty message. Now, it correctly displays the appropriate messages beside the radio buttons.

Reviewer: ```suggestion
Previously, while deleting a group, the radio buttons were displayed without a message. Now, the appropriate messages are correctly displayed beside the radio buttons.
```
